code-of-conduct.html

```diff
@@ -2,16 +2,47 @@
 layout: default
 title: Code of Conduct
 ---
-<div class="code-of-conduct container">
 
-    <div class="row">
-        <div id="mentee-section" class="col-12 section">
-            <h2>Code Of Conduct</h2>
+<div class="page-header">
+    <h1>Code of Conduct</h1>
+</div>
+<div class="container  code-of-conduct">
+    <div class="content-constrained">
+        <div class="section">
+            <p>At <b>Women Coding Community</b> we are committed to a vibrant, supportive community where women can network, share experiences,
+               and foster professional relationships, regardless of age, gender, visible or invisible disability, ethnicity, gender expression, 
+               level of experience, education, socio-economic status, nationality, personal appearance, race, religion, sexual orientation or 
+               preferred programming language(s).</p>
         </div>
-
-
-        <div class="col-12 section global-code">
-            <h2 class="text-center">TBD</h2>
+        <div class="section">
+            <h3>Our Standards:</h3>
```

Comment:
I guess these h3 should use h2 tags instead?

Reply:
I will change to h2  😎

Reply:
Screenshot with headings updated  to <h2>

![image](https://github.com/WomenCodingCommunity/WomenCodingCommunity.github.io/assets/17902969/1c5bcb22-58b3-4984-9573-766078cec525)